From 7b370ce95f879e988f197c4ef79326b34a18f55c Mon Sep 17 00:00:00 2001
From: "Bradford D. Boyle" <bradford.d.boyle@gmail.com>
Date: Sun, 24 Mar 2024 21:48:04 -0700
Subject: [PATCH 1/3] Tag pgbackrest build target as installable

By defauly, Meson does not install anything. Targets can be installed by
tagging them as installable in the build definition.

[1]: https://mesonbuild.com/Installing.html
---
 src/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/meson.build b/src/meson.build
index 3f86c8b9ae..ce95fe48f9 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -274,6 +274,7 @@ executable(
     src_pgbackrest,
     help_auto_c_inc,
     interface_auto_c_inc,
+    install : true,
     dependencies : [
         lib_backtrace,
         lib_bz2,

From e02aa05dcde772df9dbb9c83222460202a7905d8 Mon Sep 17 00:00:00 2001
From: David Steele <david@pgmasters.net>
Date: Wed, 27 Mar 2024 09:05:26 +1100
Subject: [PATCH 2/3] Move install to end.

---
 src/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/meson.build b/src/meson.build
index ce95fe48f9..f34ccb9bf4 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -274,7 +274,6 @@ executable(
     src_pgbackrest,
     help_auto_c_inc,
     interface_auto_c_inc,
-    install : true,
     dependencies : [
         lib_backtrace,
         lib_bz2,
@@ -285,5 +284,6 @@ executable(
         lib_xml,
         lib_z,
         lib_zstd,
-    ]
+    ],
+    install: true,
 )

From 86b76318d2178cda8df7f48dac0c3d2a05e22458 Mon Sep 17 00:00:00 2001
From: David Steele <david@pgmasters.net>
Date: Wed, 27 Mar 2024 09:13:29 +1100
Subject: [PATCH 3/3] Add release note.

---
 doc/xml/release/2024/2.52.xml   | 14 ++++++++++++++
 doc/xml/release/contributor.xml |  5 +++++
 2 files changed, 19 insertions(+)

diff --git a/doc/xml/release/2024/2.52.xml b/doc/xml/release/2024/2.52.xml
index 7c5df27ce7..664d3f8d6a 100644
--- a/doc/xml/release/2024/2.52.xml
+++ b/doc/xml/release/2024/2.52.xml
@@ -1,2 +1,16 @@
 <release date="XXXX-XX-XX" version="2.52dev" title="UNDER DEVELOPMENT">
+    <release-core-list>
+        <release-improvement-list>
+            <release-item>
+                <github-pull-request id="2312"/>
+
+                <release-item-contributor-list>
+                    <release-item-contributor id="bradford.boyle"/>
+                    <release-item-reviewer id="david.steele"/>
+                </release-item-contributor-list>
+
+                <p>Tag <id>pgbackrest</id> build target in meson as installable.</p>
+            </release-item>
+        </release-improvement-list>
+    </release-core-list>
 </release>
diff --git a/doc/xml/release/contributor.xml b/doc/xml/release/contributor.xml
index bfb7e4156c..793648e9a8 100644
--- a/doc/xml/release/contributor.xml
+++ b/doc/xml/release/contributor.xml
@@ -125,6 +125,11 @@
     <contributor-id type="github">bradnicholson</contributor-id>
 </contributor>
 
+<contributor id="bradford.boyle">
+    <contributor-name-display>Bradford Boyle</contributor-name-display>
+    <contributor-id type="github">bradfordboyle</contributor-id>
+</contributor>
+
 <contributor id="brent.graveland">
     <contributor-name-display>Brent Graveland</contributor-name-display>
     <contributor-id type="github">graveland</contributor-id>
